README.md

```diff
@@ -6,17 +6,17 @@ We wanted to make it open source so everyone can get visibility into what we are
 
 ### Translations
 
-Translations are done by the community. If you want to participate, feel free to join at https://poeditor.com/join/project/B4nWT6EgnD.
+Translations are done by the community. If you want to participate, feel free to join [here](https://poeditor.com/join/project/B4nWT6EgnD).
 
 ### Building
 
 - Install NodeJS v6+
 - Install Yarn
-- Run in terminal:
+- Run in terminal (Git Bash on Windows):
 	- `git submodule init`
 	- `git submodule update`
 	- `yarn`
 	- `./node_modules/.bin/gulp translations:compile`
 	- `yarn run dev`
 
-That should set up a tiny server that hosts the website for you on your computer at http://localhost:8080. Open that URL up in a web browser and you should have Game Jolt running!
+That should set up a tiny server that hosts the website for you on your computer at [localhost:8080](http://localhost:8080.) Open that URL up in a web browser and you should have Game Jolt running!
```

Comment:
The period should be outside the parentheses.

Reply:
right.